### PR TITLE
feat(cost-info-logo): added cost-info-logo opaquecircle and ellipse

### PR DESCRIPTION
**Pull request by esundqvist**

### CODE REVIEW

**Comment by robineng** [comment]
the ellipse and opaque-circle icons are looking very similar. Is it possible to only use one of them?

File: src/components/icon/icons/calculator.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  ...rest // eslint-disable-line comma-dangle
+}) {
+  const style = {
+    ...styleProp,
+    ...Icon.defaultProps.style,
+  };
+
+  return (
+    <svg width={width} height={width} style={style} {...rest} viewBox={`0 0 36 36`}>
+      <g stroke="none" strokeWidth={strokeWidth} fill="none" fillRule="evenodd">
```
Comment (bstream) [inline src/components/icon/icons/calculator.jsx]: The `transform="translate"` statements could probably be removed from the following lines (21-23).

File: src/components/icon/icons/opaque-circle.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  ...rest // eslint-disable-line comma-dangle
+}) {
+  const style = {
+    ...styleProp,
+    ...Icon.defaultProps.style,
+  };
+
+  return (
+    <svg width={width} height={height} style={style} {...rest}>
+      <g stroke="none" strokeWidth={strokeWidth} fillRule="evenodd">
```
Comment (bstream) [inline src/components/icon/icons/opaque-circle.jsx]: You can probably combine these 2 `<g>` lines, eg. change this line to: `<g stroke="none" strokeWidth={strokeWidth} fillRule="nonzero">` and then remove the line below

File: src/components/icon/icons/calculator.jsx (hunk 1 above)
Comment (esundqvist) [reply src/components/icon/icons/calculator.jsx]: They need to stay :)

Comment (bstream) [reply src/components/icon/icons/calculator.jsx]: Remove the now empty <g> tags as well

**Comment by bstream** [comment]
Did you see @robineng comment above?

**Comment by esundqvist** [comment]
@robineng the circle can't be elliptic and the ellipse can't be circular.

File: src/components/icon/icons/calculator.jsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        <g fillRule="nonzero">
+          <path d="M17.1,14.9c0,1.1-0.9,2-2,2H2c-1.1,0-2-0.9-2-2V2c0-1.1,0.9-2,2-2h13.1c1.1,0,2,0.9,2,2V14.9z" fill="#FFFFFF" />
+          <path d="M36,14.9c0,1.1-0.9,2-2,2H20.9c-1.1,0-2-0.9-2-2V2c0-1.1,0.9-2,2-2H34c1.1,0,2,0.9,2,2 L36,14.9L36,14.9z" fill="#FFFFFF" />
+          <path d="M17.1,33.6c0,1.1-0.9,2-2,2H2c-1.1,0-2-0.9-2-2V20.6c0-1.1,0.9-2,2-2h13.1c1.1,0,2,0.9,2,2 V33.6z" fill="#FFFFFF" />
+          <path
+            d="M36,33.6c0,1.1-0.9,2-2,2H20.9c-1.1,0-2-0.9-2-2V20.6c0-1.1,0.9-2,2-2H34c1.1,0,2,0.9,2,2 L36,33.6L36,33.6z"
+            fill="#FFFFFF"
```
Comment (bstream) [inline src/components/icon/icons/calculator.jsx]: `fill` should not be hardcoded to `#FFFFFF` (none of them, there are multiple occurances), rather use `fill={fill}` instead.

File: src/components/icon/icons/calculator.jsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+          />
+          <g transform="translate(4.556962, 4.050000)" fill={stroke}>
```
Comment (bstream) [inline src/components/icon/icons/calculator.jsx]: `fill={stroke}` looks a bit odd? This should probably be `fill={fill}` as well.

File: src/components/icon/icons/ellipse.jsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  };
+
+  return (
+    <svg width={width} height={height} style={style} {...rest} viewBox={`0 0 ${width} ${height}`}>
+      <ellipse
+        cx={parseInt(width / 2, 10)}
+        cy={parseInt(height / 2, 10)}
+        rx={parseInt(width / 2, 10)}
+        ry={parseInt(height / 2, 10)}
+        style={{ ...style, fill }}
```
Comment (bstream) [inline src/components/icon/icons/ellipse.jsx]: Possible to use `fill={fill}` instead of setting it in an inline style?

**Comment by robineng** [comment]
@esundqvist the ellipse looks very circular in the styleguidist build of this branch.

**Comment by bstream** [comment]
The calculator icon seems to have some anti-aliasing trouble, it looks like this in 16x16: 
image

**Pull request merged**